docker/Dockerfile

```diff
@@ -305,8 +314,16 @@ RUN PYTHON_VERSION_STR=$(echo ${PYTHON_VERSION} | sed 's/\.//g') && \
 RUN echo 'tzdata tzdata/Areas select America' | debconf-set-selections \
     && echo 'tzdata tzdata/Zones/America select Los_Angeles' | debconf-set-selections \
     && apt-get update -y \
-    && apt-get install -y software-properties-common curl sudo python3-pip \
-    && apt-get install -y ffmpeg libsm6 libxext6 libgl1 \
+    && apt-get install -y --no-install-recommends \
+        software-properties-common \
+        curl \
+        wget \
```

Comment:
Apologies, forgot to include that `wget` is also not necessary anymore. Otherwise lgtm! Dont have approver perms, but you can post in the vllm slack #pr-reviews to ask for a review!